src/data/squad.json

```
@@ -155,6 +155,11 @@
     "image": "/img/pfp/Moebius.png",
     "description": "- Decentralization and privacy maximalist.\n- In love with the mountains.\n\nFun fact: Former card magician."
   },
+  {
+    "name": "Agusduha",
+    "position": "Developer",
```

you can leave the `image` value empty
```suggestion
    "position": "Developer",
    "image": "",
```